parm/soca/obs/config/adt_3a_egm2008.yaml

```diff
@@ -3,7 +3,7 @@ obs space:
   obsdatain:
     engine:
       type: H5File
-      obsfile: !ENV ${DATA}/obs/${OPREFIX}adt_3a_egm2008.${PDY}${cyc}.nc4
+      obsfile: !ENV ${COM_OBS}/${OPREFIX}adt_3a_egm2008.${PDY}${cyc}.nc4
```

Comment:
So do you not copy/link the obs to a runtime directory?
Generally speaking I think our convention has been to copy/link from `$COM_OBS` to `$DATA` as part of the init step

Reply:
... Looks like somebody took a short cut! I should have kept that ```DATA/obs``` dir ... 
I'll revert the yamls and copy or link from ```COM_OBS``` to  ```${DATA}/obs```, but with all the obs.